### PR TITLE
yt-dlp: add page

### PR DESCRIPTION
**Pull request by n14s**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

**Review by blueskyson — APPROVED** [review]

File: pages/common/yt-dlp.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+> A youtube-dl fork with additional features and fixes.
+> Download videos from YouTube and other websites.
+> More information: <https://github.com/yt-dlp/yt-dlp>.
+
+- Download a video or playlist (with the default options from command below):
+
+`yt-dlp "{{https://www.youtube.com/watch?v=oHg5SJYRHA0}}"`
+
+- Download best format that contains video, and if it doesn't already have an audio stream, merge it with best audio-only format (Default):
```
Comment (CleanMachine1) [inline pages/common/yt-dlp.md:11]: Could you please reword this. I have almost no idea what it means, do you mean to say

- Download a video with the highest video and sound quality:

Comment (n14s) [reply pages/common/yt-dlp.md]: @navarroaxel I understand @CleanMachine1 confusion, I was confused myself haha.
I fixed the explanation.

File: .gitpod.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# This configuration file was automatically generated by Gitpod.
```
Comment (navarroaxel) [inline .gitpod.yml:1]: @n14s please remove this file.

Comment (n14s) [reply .gitpod.yml]: done

**Review by Waples — APPROVED** [review]
Very nice!

**Pull request merged**